src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -74,25 +72,6 @@ internal partial class SymbolTreeInfo : IChecksummedObject
 
         private readonly SpellChecker _spellChecker;
 
-        private static readonly StringSliceComparer s_caseInsensitiveComparer =
-            StringSliceComparer.OrdinalIgnoreCase;
```

Comment:
this code originally worked by having a single massive alloc for all the unique strings concatenated.  it then passed around spans inside that string.  we undid that and just pointed back to small strings.  So all these StringSlice helpers aren't needed anymore.